app/controllers/users/verify_account_controller.rb

```diff
@@ -34,7 +34,13 @@ def create
         analytics.track_event(Analytics::ACCOUNT_VERIFICATION_SUBMITTED, result.to_h)
 
         if result.success?
-          create_user_event(:account_verified)
+          event = create_user_event_with_disavowal(:account_verified)
+          UserAlerts::AlertUserAboutAccountVerified.call(
+            user: current_user,
+            date_time: event.created_at,
+            app: nil,
```

Comment:
My hunch was we wouldn't, since it would be however long it takes to receive the letter and enter the code, but there isn't any harm in adding it. Included in 6e0caa3